.github/workflows/rust-publish-dry-run-v2.yml

```
@@ -30,16 +30,26 @@ jobs:
     env:
       CARGO_BUILD_TARGET: ${{ inputs.target }}
       CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
+      RUST_BACKTRACE: 1
     steps:
-    - uses: actions/checkout@v3
-
+    - uses: actions/checkout@v4
     - uses: stellar/actions/rust-cache@main
 
     - run: rustup update
     - run: rustup target add ${{ inputs.target }}
-    - if: inputs.target == 'aarch64-unknown-linux-gnu'
+
+    - name: Install add'l compilers (Linux arm64 only)
+      if: inputs.target == 'aarch64-unknown-linux-gnu'
       run: sudo apt-get update && sudo apt-get -y install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
 
+    - name: Update GNU Make (macOS only)
+      if: inputs.target == 'x86_64-apple-darwin' || inputs.target == 'aarch64-apple-darwin'
+      run: |
+        which make
+        make --version
+        brew update && brew install make
+        echo "/usr/local/opt/make/libexec/gnubin/" >> $GITHUB_PATH
```

It's safer to use `brew --prefix` to figure out what the brew prefix is on the machine. It isn't consistent between all targets or mac versions even I think.

So I think this could be changed to:
```suggestion
        echo "$(brew --prefix)/opt/make/libexec/gnubin/" >> $GITHUB_PATH
```

---

nice idea, thanks!